<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>


  <title>科技大學智慧型系統實驗室 研究論文 NTUST Fuzzy Neural Lab Paper</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf8">
</head>

<body background="/picture/background/gray_roc.gif" bgcolor="#ffffff" link="#55ffff" text="#000000" vlink="#6966ff">

<img src="/picture/logo/logo.gif" align="left"> <strong> <font size="5">國立台灣科技大學 資訊工程系所</font><br>

<font size="6">智慧型系統實驗室 研究論文</font><br>

<font size="4">Intelligent System Laboratory Paper</font> </strong> 
<hr size="10">
<h1>95級畢業碩士 羅百玲 發表論文</h1>

<hr size="5">
<center>
  
<table width="80%">

    <tbody>
    <tr>

      <td> 
        
      <center>
		
      <h3><a name="AIC">一種非監督式樣型辨識方法尋找辨認調控因子結合位--植基於DNA微小序列片段特徵之研究&nbsp;</a></h3>

        </center>

        
      <h4>摘要</h4>

        
      <p align="justify"> &nbsp;&nbsp;&nbsp;
大部分的調控因子結合位都位在基因的上游序列中，為了要瞭解基因被調控的機制，辨認出這些調控因子的結合位，是這個任務的首要工作。然而，最近針對辨認調
控因子結合位的方法所發表的評量顯示，這項工作仍然非常具有挑戰性，特別是較高等與複雜的生物，例如人類。這項工作受限於調控機制不明，因此沒有一個確切
的表達方式，來描述同一個調控因子的結合位，除此之外，上游序列中只有很小一部分的基因體會與調控因子結合，所以這項工作也受限於巨大的背景雜訊，因此這
項工作至今仍然是一項重要而難解的生物問題。<br>
在本篇論文當中，我們提出了一個非監督式樣型辨識方法，來處理這種含有大量的背景雜訊以及並非所有結合位都已被發現的不完全(incomplete)且不
平均(unbalanced)生物資料。為了要模擬基因上游序列中每個片段結合的活性，我們提出了利用片段中的短序列之特徵以及其頻率的特徵向量。當特徵
向量被群聚分群在不同的節點上時，為了要辨認出可能含有結合位的節點，我們考慮了每個節點的過度表現性以及在每個序列的分佈情形。為了要評估我們方法的表
現，我們採用了最近一篇針對辨認調控因子結合位的計算方法做出評估的期刊論文所提供的基準實驗資料。經由實驗結果顯示，我們所提出的方法，與相關的類似方
法(SOMBRERO)比較，較能將有可能是結合位的樣型在在較前面的名次預測出，表示我們所提出的結合位的表示方式，以及排序的方法，使預測調控因子結
合位時較有效率。
        </p>

     </td>

    </tr>

  
  </tbody>
</table>

  
<hr>
  
<table width="80%">

    <tbody>
    <tr>

      <td> 
        
      <center>
		
      <h3><a name="AIE">An Unsupervised Pattern Recognition Method for Identifying TFBS Based on DNA Short Sequence Features</a></h3>

        </center>

	
      <h4>Abstract</h4>

	
      <p align="justify">&nbsp;&nbsp;&nbsp; Identifying binding sites
for the transcription factor in the upstream sequences of genes to
which the factor binds is the first step to understand the gene
regulatory mechanism. Recent assessment of computational tools for
identifying these binding sites indicates that identifying these
regulatory elements remains a challenging task in higher organisms,
such as the human species. The task is limited in the intrinsic
subtlety of binding sites and the huge background noise. That is, only
a small portion of genome will be bound by transcription factors and
sequence-specific recognition for binding is subtle. <br>
In this thesis, we proposed an unsupervised pattern recognition method
to handle the incomplete and unbalanced biological data. To model the
binding activity, a vector of small sequence features was proposed. To
identify candidate pattern for binding sites, the overall
over-representative and sequence popularity of each pattern are taken
into consideration in ranking. To evaluate the performance and to
compare with related work, a benchmark which has been used to assess
existing tools was adopted. The experimental results show that the
proposed methodology outperforms the related work in terms of the
nucleotide level performance when we only consider the top 3 nodes in
the ranking. The proposed representation of binding sites and the
ranking mechanism make efficient predictions of binding sites.<br>

     </p>
      </td>

    </tr>

  
  </tbody>
</table>


  <br>

</center>

<hr size="6">
<center>
  <a href="/index.html"><img src="/picture/icons/home.gif" border="0"></a> <a href="mailto:WWW@neuron.et.ntust.edu.tw"> 
  <img src="/picture/icons/email.gif" border="0"></a> 
</center>

</body>
</html>
